mRides-app/Tasks/FindMatchAsyncTask.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Runtime;
+using Android.Views;
+using Android.Widget;
+using Java.Lang;
+
+using mRides_app.Models;
+using static mRides_app.Models.Request;
+using mRides_app.Mappers;
+using mRides_app.Tasks.Callbacks;
+
+namespace mRides_app.Tasks
+{
+    public class FindMatchAsyncTask : AsyncTask<Java.Lang.Void, Java.Lang.Void, Java.Lang.Void>
+    {
+        // Data necessary to build the request for the match
+        private Request userRequest;
+
+        // Object to call once the task is complete
+        private IOnFindMatchCompleteCallback callBackObject;
+
+        // List of requests matched
+        List<Request> listOfMatchedRequests;
+
+        // Mapper used to make the request
+        private ConsoleMapper consoleMapper;
+        
+        /// <summary>
+        /// 
```

Comment:
Don't forget to write a description of each method inside this class along with the parameters :) They are all blank